modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -54,5 +63,65 @@
     $("[name=cleardef]").blur();
   });
   <{/if}>
+  
+  <{if $content.type eq "select"}>
+  checkNAMES();
+  /**
+   *this function will consantly check for {USERNAMES} and {FULLNAMES}
+   */
+  function checkNAMES(){
+    var x=0,y=-1,z=-1;    //y for {FULLNAMES}  z for {USERNAMES}
+    for(x=0;x<length;x++){

```

Comment:
A loop is not needed here, because {USERNAMES} or {FULLNAMES} must be the first and only option.